lib/Event.php

```diff
@@ -902,7 +902,16 @@ public function toiCalendar($calendar, $includeFiles = true)
                 }
             }
 
-            $vEvent->setAttribute($attribute, $email, $params);
+            /**
+             * Do not set attributes without a name
+             * This may happen if an (imported?) attendee has no proper email
+             * address AND is not a horde user.
+             * 
+             * See also https://lists.horde.org/archives/kronolith/Week-of-Mon-20210125/009723.html
+             */
+            if ($attribute) {
```

Comment:
We should perform the existence check using !empty($attribute) to avoid undefined error notices.  Otherwise, the patch looks good, thanks!